src/webgl/p5.Camera.js

```diff
@@ -1720,23 +1720,19 @@ p5.Camera.prototype._orbit = function(dTheta, dPhi, dRadius) {
   let camTheta = Math.atan2(diffX, diffZ); // equatorial angle
   let camPhi = Math.acos(Math.max(-1, Math.min(1, diffY / camRadius))); // polar angle
 
-  // add change
-  camTheta += dTheta;
-  camPhi += dPhi;
-  camRadius += dRadius;
+  // add change according to the direction of this.upY
+  let directionflag = this.upY > 0 ? 1 : -1;
```

Comment:
Maybe we can name this variable `newUpY` since that's what it will end up getting used for?

And then the comment about changing according to the direction of `this.upY` would maybe then be better placed at the if statement where we flip it, with a small explanation of *why* we have to flip it (something about orbiting to the other side and now being upside-down.)